ftd/built-in-types.ftd

```diff
@@ -20,53 +20,53 @@ These types can be categorized into two main groups: **Primitive** and
 -- ds.h1: Contents
 
 
-- [Primitive Types](built-in-types#primitive-types)
-  - [`boolean`](built-in-types#boolean)
-  - [`integer`](built-in-types#integer)
-  - [`decimal`](built-in-types#decimal)
-  - [`string`](built-in-types#string)
-  - [`caption`](built-in-types#caption)
-  - [`body`](built-in-types#body)
-  - [`caption or body`](built-in-types#caption-or-body)
-  - [`ftd.ui`](built-in-types#ftd-ui)
-  - [`children`](built-in-types#children)
-- [Derived Types](built-in-types#derived-types)
-  - [`ftd.align-self`](built-in-types#ftd-align-self)
-  - [`ftd.align`](built-in-types#ftd-align)
-  - [`ftd.anchor`](built-in-types#ftd-anchor)
-  - [`ftd.linear-gradient`](built-in-types#ftd-linear-gradient)
-  - [`ftd.linear-gradient-color`](built-in-types#ftd-linear-gradient-color)
-  - [`ftd.breakpoint-width-data`](built-in-types#ftd-breakpoint-width-data)
-  - [`ftd.linear-gradient-directions`](built-in-types#ftd-linear-gradient-directions)
-  - [`ftd.background-image`](built-in-types#ftd-background-image)
-  - [`ftd.background-position`](built-in-types#ftd-background-position)
-  - [`ftd.background-repeat`](built-in-types#ftd-background-repeat)
-  - [`ftd.background-size`](built-in-types#ftd-background-size)
-  - [`ftd.background`](built-in-types#ftd-background)
-  - [`ftd.border-style`](built-in-types#ftd-border-style)
-  - [`ftd.color`](built-in-types#ftd-color)
-  - [`ftd.display`](built-in-types#ftd-display)
-  - [`ftd.color-scheme`](built-in-types#ftd-color-scheme)
-  - [`ftd.cursor`](built-in-types#ftd-cursor)
-  - [`ftd.image-src`](built-in-types#ftd-image-src)
-  - [`ftd.length`](built-in-types#ftd-length)
-  - [`ftd.length-pair`](built-in-types#ftd-length-pair)
-  - [`ftd.loading`](built-in-types#ftd-loading)
-  - [`ftd.overflow`](built-in-types#ftd-overflow)
-  - [`ftd.region`](built-in-types#ftd-region)
-  - [`ftd.resize`](built-in-types#ftd-resize)
-  - [`ftd.resizing`](built-in-types#ftd-resizing)
-  - [`ftd.responsive-type`](built-in-types#ftd-responsive-type)
-  - [`ftd.shadow`](built-in-types#ftd-shadow)
-  - [`ftd.mask`](built-in-types#ftd-mask)
-  - [`ftd.spacing`](built-in-types#ftd-spacing)
-  - [`ftd.text-align`](built-in-types#ftd-text-align)
-  - [`ftd.text-input-type`](built-in-types#ftd-text-input-type)
-  - [`ftd.text-style`](built-in-types#ftd-text-style)
-  - [`ftd.text-transform`](built-in-types#ftd-text-transform)
-  - [`ftd.type`](built-in-types#ftd-type)
-  - [`ftd.white-space`](built-in-types#ftd-white-space)
-  - [`ftd.type-data`](built-in-types#ftd-type-data)
+- [Primitive Types](/built-in-types#primitive-types)
```

Comment:
```suggestion
- [Primitive Types](/built-in-types/#primitive-types)
```

Reply:
That extra slash was causing the reload issue when it was already an intra-link.